src/components/WorkSchedule/ColumnRender.js

```diff
@@ -2,14 +2,16 @@ import React, { Component } from "react";
 // import { data }   from '../../data/Data_WorkSchedule';
 // @TODO as we commented that data, should we get rid of it?
 // or maybe we can use it as readme sample about what kind of datab with what structure should have our servercall?
-import data from "@groceristar/groceristar-fetch/mealCalendar";
+// import data from "@groceristar/groceristar-fetch/mealCalendar";
 // import { rendererFunc } from "./Methods.js";
 import RenderData from "./RenderData.js";
+import Selector from "../selectors/selector.js";
```

Comment:
I think I should delete declaration of class
When I use this It's doesn't work 
```
import data from "@groceristar/groceristar-fetch/mealCalendar";

class Selector {
 function getDishByWeek(randomWeek){
    return data.getDishByWeek(randomWeek);
  }
}
export {Selector.prototype.getDishByWeek}
```
When I execute this, It's working fine
```
import data from "@groceristar/groceristar-fetch/mealCalendar";

 function getDishByWeek1(randomWeek){
    return data.getDishByWeek(randomWeek);
  }

export {getDishByWeek1}
```

Reply:
maybe  `export Selector;` will work?

Reply:
and sure, you can delete a class, if you think it's not work well.
because i also don't like that we create a method, with name of method from fetch plugin.
it looks not ideal..

i'm ok if you'll just create an object
```
export {
getDishByWeek: data.getDishByWeek
}
```

Reply:
It's Syntax from CommonJS? 
But I need to give parameters

Reply:
Hmm, i don't know the names, aka CommonJS :)

check this file, method and how it exported: https://github.com/GroceriStar/calendar/blob/master/src/components/WorkSchedule/Methods.js#L50
i think it'll give you an answer

Reply:
In 1.1.28 version of fetch function getRecipe() return udefind

Reply:
then please create an issue at fetch-github, then create a card at trello and mention Audax - because he made that changes